src/com/sap/piper/analytics/InfluxData.groovy

```diff
@@ -38,4 +38,29 @@ class InfluxData implements Serializable{
     public static void reset(){
         instance = null
     }
+
+    public static void readFromDisk(script) {
+
+        def influxValues = script.findFiles(glob: '.pipeline/influx/*')
+
+        influxValues.each({f ->
+            def fileName = f.getName()
+            def parts = fileName.split('.pipeline/influx/')[1].split('/')
```

Comment:
I don't understand what is happening here. The single glob-star `*` should match only entries in the sub-folder .pipeline/influx/, not in any sub-folders. So the second array-entry from `fileName.split()` should not contain any slashes.

From reading the changes in the generated code, should the glob-pattern be '.pipeline/influx/**'?

Reply:
Good hint.